languages.toml

```diff
@@ -387,6 +387,13 @@ comment-token = "#"
 indent = { tab-width = 2, unit = "  " }
 language-server = { command = "cmake-language-server" }
 
+[[language]]
+name = "make"
+scope = "source.make"
+file-types = ["Makefile"]
```

Comment:
Could we also add the other variants, `makefile` and `GNUMakefile`? And also could we throw in `justfile` and `.justfile`?

Reply:
Good thinking!

Reply:
Wait, I thought there is tree-sitter-just? https://github.com/IndianBoy42/tree-sitter-just

Reply:
Oh interesting. Is it good to use? It mentions it's a work in progress and doesn't have tests yet.